### PR TITLE
[WIP] Dashboard

### PR DESCRIPTION
**Pull request by NJichev**
Created dashboad, currently working on it's ajax.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -29,6 +29,32 @@ def test_numbers_arent_odd_when_they_are_like_2
...
+
+  class PrimeTest < Minitest::Test
+    def test_numbers_are_prime_when_they_are_like_3_and_2_and_17_and_97
+      assert_equal true, prime?(3)
+      assert_equal true, prime?(2)
+      assert_equal true, prime?(17)
+      assert_equal true, prime?(97)
+    end
+
+    def test_numbers_arent_prime_when_they_are_like_16_100_-100_0_1
```
Comment (gsamokovarov) [inline db/seeds.rb]: That's not valid Ruby, right 😅

Comment (NJichev) [inline db/seeds.rb]: What seems to be the problem?

Comment (gsamokovarov) [inline db/seeds.rb]: The dash (`-`) in a Ruby identifier name.

Comment (NJichev) [inline db/seeds.rb]: Facepalmed myself.

**Comment by NJichev** [comment]
What would be a good approach for the ajax on the dashboard and could we use ActionCable there?

**Comment by gsamokovarov** [comment]
If you wanna use Action Cable there, go for it.

**Pull request merged**